src/components/nav-logo/nav-logo.scss

```diff
@@ -0,0 +1,54 @@
+:host {
+  @apply inline-flex outline-none;
+  & a {
+    @apply flex m-0 items-center justify-center cursor-pointer transition-default focus-base no-underline;
+    border-block-end: 2px solid transparent;
+  }
+  & img {
+    @apply flex h-7 m-0;
+  }
+}
+
+a:hover,
```

Comment:
nitpicky, but maybe we should use a class for these instead of an element tag. What if this component adds another anchor tag in its shadowRoot.

Reply:
Same with `img`

Reply:
Applies to other CSS files as well.

Reply:
Since this component is very basic, may be we can add a class when there is a requirement to add more anchor element or img.